NuKeeper.Gitea/GiteaPlatform.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using NuKeeper.Abstractions;
+using NuKeeper.Abstractions.CollaborationModels;
+using NuKeeper.Abstractions.CollaborationPlatform;
+using NuKeeper.Abstractions.Configuration;
+using NuKeeper.Abstractions.Logging;
+
+namespace NuKeeper.Gitea
+{
+    public class GiteaPlatform : ICollaborationPlatform
+    {
+        private readonly INuKeeperLogger _logger;
+        private GiteaRestClient _client;
+
+        public GiteaPlatform(INuKeeperLogger logger)
+        {
+            _logger = logger;
+        }
+
+        public void Initialise(AuthSettings settings)
+        {
+            var httpClient = new HttpClient
+            {
+                BaseAddress = settings.ApiBase
+            };
+
+            _client = new GiteaRestClient(httpClient, settings.Token, _logger);
+        }
+
+        public async Task<User> GetCurrentUser()
+        {
+            var user = await _client.GetCurrentUser();
+            return new User(user.Login, user.FullName, user.Email);
+        }
+
+        public async Task OpenPullRequest(ForkData target, PullRequestRequest request, IEnumerable<string> labels)
+        {
+            var projectName = target.Owner;
+            var repositoryName = target.Name;
+
+            var mergeRequest = new Model.CreatePullRequestOption
+            {
+                Title = request.Title,
+                Head = request.Head,
+                Body = request.Body,
+                Base = request.BaseRef,
+                Labels = new long[] { 0 },
+
+                // Gitea needs a due date for its pull requests. Maybe get this from the config file?
+                DueDate = DateTime.Now.AddDays(7)
+            };
+
+            await _client.OpenPullRequest(projectName, repositoryName, mergeRequest);
+        }
+
+        public async Task<IReadOnlyList<Organization>> GetOrganizations()
+        {
+            var orgas = await _client.GetOrganizations();
+            return orgas?.Select(x => MapOrganization(x)).ToList() ?? new List<Organization>();
+        }
+
+        public async Task<IReadOnlyList<Repository>> GetRepositoriesForOrganisation(string organizationName)
+        {
+            var repos = await _client.GetOrgaRepositories(organizationName);
```

Comment:
Type in `GetOrga`? Should be `GetOrganisationRepositories`?

Reply:
should be
```suggestion
            var repos = await _client.GetOrganisationRepositories(organizationName);
```

Reply:
Could you update that?